supplementary_style_guide/glossary_terms_conventions/general_conventions/v.adoc

```diff
@@ -263,6 +274,39 @@ Always use "VM Portal" and capitalize the product name.
 
 *See also*: xref:persistent-storage[persistent storage]
 
+[discrete]
+[[volume-file]]
+==== image:images/yes.png[yes] volume file (noun)
+*Description*: In Red Hat Gluster Storage, a configuration file that determines the behavior of the trusted storage pool.
+
+*Use it*: yes
+
+*Incorrect forms*: volfile
+
+*See also*:
+
+[discrete]
+[[volume-group]]
+==== image:images/yes.png[yes] volume group (noun)
+*Description*: A combined group of physical volumes, which creates a pool of disk space out of which logical volumes can be allocated.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[worker-node]]
+==== image:images/yes.png[yes] worker node (noun)
+*Description*: In OpenShift, a worker node runs containerized applications.
```

Comment:
```suggestion
*Description*: In Red Hat OpenShift, a _worker node_ runs containerized applications.
```

However, I think we actually prefer the term "compute node". We do have an entry for this already, so at the very least, we should add xrefs between the two, or we should just drop this entry.